schemas/README.md

```diff
@@ -1,13 +1,122 @@
+# Schema files for description of simulation model parameters and input data
 
-This directory contains a descriptions of all simulation model parameters and of all input data required for the derivation of simulation model parameters.
+This directory contains a descriptions of all simulation model parameters and of all input data required for the derivation and setting of simulation model parameters.
+The parameter description includes (among others fields) name, type, format, applicable telescopes, and parameter description.
 
-The files are in [json-schema](https://json-schema.org/) and include (among others) name, type, format, applicable telescopes, and parameter description.
+The files are in human readable yaml format and follow a fixed [json-schema](https://json-schema.org/).
+The full description of the schema is in [jsonschema.yml](./jsonschema.yml), including plenty of comments.
 
-Files and directories:
-- [schema/](./): reference schemas for all model parameters (one file per parameter) and input data required by the derivation tools
-- [schema/jsonschema.yml](./jsonschema.yml): json schema for validation of all schema files
+## Typical Example
 
-## Technical notes
+The following example is typical for a single parameter description.
 
-For easy generation of json schema files from yaml example, see https://codebeautify.org/yaml-to-json-schema-generator
+```yaml
+%YAML 1.2
+---
+title: Schema for num_gains model parameter
+version: 0.1.0
+base_schema: simpipe-schema
+base_schema_url: https://github.com/gammasim/workflows/schema/jsonschema.yml
+base_schema_version: 0.1.0
+name: num_gains
+description: Number of different gains the input signal gets digitized.
+data:
+  - type: int
+    units: dimensionless
+    default: 1
+    allowed_range:
+      min: 1
+      max: 2
+instrument:
+  class: Camera
+  type:
+    - LSTCam
+    - FlashCam
+    - NectarCam
+    - SSTCam
+    - SCTCam
+activity:
+  setting:
+    - SetParameterFromExternal
+  validation:
+    - ValidateParameterByExpert
+source:
+  - Initial instrument setup
+simulation_software:
+  - name: sim_telarray
+```
+
+## Valid Keys
+
+Valid keys are described in detail in [jsonschema.yml](./jsonschema.yml). The following list gives a short overview.
+
+### Header section
+
+- `title`: Title of the schema file.
+- `version`: Version of this schema file.
+- `base_schema`: Name of the base schema.
+- `base_schema_url`: URL of the base schema.
+- `base_schema_version`: Version of the base schema.
+
+### Parameter description
+
+- `name`: Name of the parameter.
+- `description`: Description of the parameter.
+- `short_description`: Short description of the parameter.
+
+### Parameter data
+
+The *data* field is used to describe the actual type, format, and allowed values of the parameter.
+
+- `type`: Data type of the parameter.
+- `units`: Units of the parameter (compatible with astropy units). Note that units are explicitly listed in the jsonschema.yml file.
```

Comment:
what does compatible with astropy units mean? Is it that if you pass it as an argument to`astropy.Units()` it will give you an instance of `astropy.Quantity` ? If yes, then please include this explanation.

Reply:
It means that astropy units can use these units. How and which functions are called is not so important in this context - isn't `compatible` saying exactly this?